mula/scheduler/context/context.py

```diff
@@ -99,7 +99,7 @@ def __init__(self) -> None:
                     structlog.dev.set_exc_info,
                     structlog.stdlib.PositionalArgumentsFormatter(),
                     structlog.processors.TimeStamper("iso", utc=False),
-                    structlog.dev.ConsoleRenderer(),
+                    structlog.dev.ConsoleRenderer(colors=False),
```

Comment:
Any particular reason why this is disabled? In the other modules, when the `text` log format is selected we do colorize the log records